frontend/src/metabase/css/query_builder.module.css

```diff
@@ -4,178 +4,177 @@
   max-height: inherit;
 }
 
-:global(.SelectionItem) {
+.SelectionItem {
   display: flex;
   align-items: center;
   cursor: pointer;
   padding: 0.75rem 1.5rem 0.75rem 0.75rem;
   background-color: var(--color-bg-white);
 }
 
-:global(.SelectionItem:hover) {
+.SelectionItem:hover {
   background-color: currentColor;
 }
 
-:global(.SelectionItem .Icon) {
+.SelectionItem :global(.Icon) {
   margin-left: 0.5rem;
   margin-right: 0.75rem;
   color: currentcolor;
 }
 
-:global(.SelectionItem .Icon-check) {
+.SelectionItem :global(.Icon-check) {
   opacity: 0;
 }
 
-:global(.SelectionItem .Icon-chevrondown) {
+.SelectionItem :global(.Icon-chevrondown) {
   opacity: 1;
 }
 
-:global(.SelectionItem:hover .Icon) {
+.SelectionItem:hover :global(.Icon) {
   color: var(--color-text-white) !important;
 }
 
-:global(.SelectionItem:hover .SelectionModule-display) {
+.SelectionItem:hover .SelectionModuleDisplay {
   color: var(--color-text-white);
 }
 
-:global(.SelectionItem:hover .SelectionModule-description) {
+.SelectionItem:hover .SelectionModuleDescription {
   color: var(--color-text-white);
 }
 
-:global(.SelectionItem.SelectionItem--selected .Icon-check) {
+.SelectionItem.SelectionItemSelected :global(.Icon-check) {
   opacity: 1;
 }
 
-:global(.SelectionModule-display) {
+.SelectionModuleDisplay {
   color: currentColor;
   margin-bottom: 0.25em;
 }
 
-:global(.SelectionModule-description) {
+.SelectionModuleDescription {
   color: var(--color-text-medium);
   font-size: 0.8rem;
 }
 
-:global(.Visualization) {
+.Visualization {
   transition: background 0.3s linear;
 }
 
-:global(.Visualization.Visualization--loading) {
+.Visualization.VisualizationLoading {
   transition: background 0.3s linear;
 }
 
-:global(.Visualization.Visualization--error) {
-  justify-content: center;
-}
-
-:global(.Loading) {
+.Loading {
   background-color: color-mod(var(--color-bg-white) alpha(-28%));
   transition: opacity 0.5s;
 }
 
-:global(.Loading.Loading--hidden) {
+.Loading.LoadingHidden {
   background-color: transparent;
 }
 
-:global(.QueryError) {
+.QueryError {
   flex-direction: column;
   justify-content: center;
   max-width: 500px;
   margin-left: auto;
   margin-right: auto;
 }
 
-:global(.QueryError-image) {
+.QueryErrorImage {
   background-repeat: no-repeat;
   margin-bottom: 1rem;
 }
 
-:global(.QueryError-image--noRows) {
+.QueryErrorImageNoRows {
   width: 120px;
   height: 120px;
   background-image: url("~assets/img/no_results.svg");
 }
 
-:global(.QueryError-image--queryError) {
+.QueryErrorImageQueryError {
   width: 120px;
   height: 120px;
   background-image: url("~assets/img/no_understand.svg");
 }
 
-:global(.QueryError-image--serverError) {
+.QueryErrorImageServerError {
   width: 120px;
   height: 148px;
   background-image: url("~assets/img/blown_up.svg");
 }
 
-:global(.QueryError-messageText) {
+.QueryErrorImageTimeout {
+  width: 120px;
+  height: 120px;
+  background-image: url("~assets/img/stopwatch.svg");
+}
```

Comment:
Where is this rule from? I couldn't find we use `stopwatch.svg` in master.

Reply:
Yeah, that makes sense. I meant `stopwatch.svg` wasn't used/referenced in the code, because it was removed unintentionally.

Reply:
One of the lessons I learned all these years is to not use dynamic strings. That a recipe for disasters 😆